WalletWasabi.Gui/Shell/Commands/DiskCommands.cs

```diff
@@ -61,6 +77,11 @@ private void OnOpenConfigFile()
 			IoHelpers.OpenFileInTextEditor(Global.Config.FilePath);
 		}
 
+		private void OnFileOpenException(Exception ex)
+		{
+			Logging.Logger.LogError<DiskCommands>(ex);
```

Comment:
Should warn the user in a toast regarding the error.
related: https://github.com/zkSNACKs/WalletWasabi/issues/845